tests/publiccloud/install_ipa.pm

```diff
@@ -0,0 +1,70 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install IPA tool
+#
+# Maintainer: Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
+use registration 'add_suseconnect_product';
+use serial_terminal 'select_virtio_console';
+use version_utils qw(is_sle is_opensuse);
+
+
+sub run {
+    my ($self) = @_;
+
+    select_virtio_console();
+
+    if (is_sle) {
+        my $modver = get_required_var('VERSION') =~ s/-SP\d+//gr;
+        add_suseconnect_product('sle-module-public-cloud', $modver);
+    }
+    zypper_call('ar ' . get_required_var('PUBLIC_CLOUD_TOOLS_REPO'));
+    zypper_call('--gpg-auto-import-keys in python3-ipa python3-ipa-tests');
+
+    # WAR install awscli from pip instead of using the package bcs#1095041
+    zypper_call('in gcc python3-pip');
+    if (is_opensuse) {
+        zypper_call('in python3-devel');
+        assert_script_run("pip3 install pycrypto");
+    }
+    assert_script_run("pip3 install awscli");
+    assert_script_run("pip3 install keyring");
+
+    # Create some folders, ipa will need them
```

Comment:
not relevant, but I think in Linux it's called 'directory' :)

Reply:
yeah, thx for that :-)